components/script/dom/htmllinkelement.rs

```diff
@@ -259,6 +299,109 @@ impl VirtualMethods for HTMLLinkElement {
 }
 
 impl HTMLLinkElement {
+    /// <https://html.spec.whatwg.org/multipage/semantics.html#create-link-options-from-element>
+    fn processing_options(&self) -> LinkProcessingOptions {
+        // Step 1.
```

Comment:
Let's add the spec text for each step, since the step numbering often drifts over time.